docs/advanced/subclassed-layers.md

```diff
@@ -7,14 +7,13 @@ the vertex and/or fragment shaders.
 If a small feature is missing from a layer, subclassing can often be a
 good technique to add it.
 
-## Overriding Attribute Calculation
+### Overriding Attribute Calculation
```

Comment:
Apparently @apercu's concern is that CSS only works on demo site. His task is to make this look good when read directly on github.

Still what stops github from changing their formatting at any time. We'd need to change our documentation structure if they do. Better to go with 1,2,3 than 1,3,5 and accept that github docs won't look as good as they could.